docs/usage.html

```diff
@@ -22,7 +22,7 @@
         <a class="header-link" href="index.html">
           Frames 3.5.3 Docs
         </a>
-         (50% documented)
```

Comment:
Looks like doc coverage went down a lot. Might make sense since we've added a ton of internal code, but may need to review if this is a metric we need to consider or not